.github/workflows/continuous-delivery.yml

```diff
@@ -7,6 +7,11 @@ on:
         description: Merge staging into master first? (y/N)
         required: false
         default: 'n'
```

Comment:
Can you make this one boolean too? I think I made it like this because boolean was not an option back in the day, but I see the UI nicely supports it now. Don't forget the checks below :)